spec-tests/src/test/java/org/candlepin/spec/HypervisorCheckInSpecTest.java

```diff
@@ -146,45 +111,42 @@ private void initConsumersAndPools() {
         PoolDTO pool2 = Pools.random(virtLimitProduct);
         ownerApi.createPool(owner.getKey(), pool2);
 
+        ApiClient hostConsumerClient = ApiClients.ssl(hostConsumer);
         hostConsumerClient.consumers().bindPool(hostConsumer.getUuid(), pool1.getId(), 1);
-        virtLimitPool = getVirtLimitPool(ownerApi.listOwnerPools(owner.getKey()));
+        return getVirtLimitPool(ownerApi.listOwnerPools(owner.getKey()));
+    }
 
-        guestConsumer1 = Consumers.random(owner)
-            .name(StringUtil.random("guest"))
-            .type(ConsumerTypes.System.value())
-            .facts(Map.of("virt.uuid", guest1VirtUuid, "virt.is_guest",
-                "true", "uname.machine", "x86_64"));
-        guestConsumer1 = consumerApi.createConsumer(guestConsumer1);
-        guestConsumer2 = Consumers.random(owner)
-            .name(StringUtil.random("guest"))
-            .type(ConsumerTypes.System.value())
-            .facts(Map.of("virt.uuid", guest2VirtUuid, "virt.is_guest",
-                "true", "uname.machine", "x86_64"));
-        guestConsumer2 = consumerApi.createConsumer(guestConsumer2);
+    private void setupOwnerUserClient() {
+        owner = ownerApi.createOwner(Owners.random());
+        UserDTO user = UserUtil.createUser(client, owner);
+        userClient = ApiClients.trustedUser(user.getUsername());
```

Comment:
Can we replace all `ApiClients.trustedUser(username)` with `ApiClients.basic(username, password)` through out this PR (and for all other spec tests from now on)? The ApiClients.trustedUser and ApiClients.trustedConsumer should never be used in spec tests (aside from the few tests that explicitly test those authentication methods, e.g. TrustedConsumerAuthSpecTest & TrustedUserAuthSpecTest).